src/eduid_idp/tests/test_actions.py

```diff
@@ -82,7 +82,7 @@ def setUp(self):
         # setup some test data
         _email = 'johnsmith@example.com'
         self.test_user = self.amdb.get_user_by_mail(_email)
-        self.test_action = self.actions.add_action(userid = self.test_user.user_id,
+        self.test_action = self.actions.add_action(self.test_user.eppn,
```

Comment:
Same here - does it make more sense to pass in the whole user?

Reply:
Same answer - add_action will look into the form of the arg to decide what kind of action (old or new) it adds